tfx/components/transform/executor.py

```diff
@@ -652,7 +679,7 @@ def to_runner_api_pickled(self, context):
       # picklable.
       return self.to_runner_api_parameter(context)
 
-    def expand(
+    def expand( # pylint: disable=arguments-differ
```

Comment:
two space before, let's not add it, I feel internal doesn't care about this lint

Reply:
ok i removed this pylint